core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
     return findContext(contextName) != null;
   }
 
+  /* (non-Javadoc)
```

Comment:
I think the formatter which was in there at least once was removed.  If that was back it could enforce line endings at least.  I also noticed on a newer build of mars.1 that I got where initially the jautodoc didn't load that once I got a patched copy it seems to not add these non javadoc entries now yet they still behave the same.  So I want to research a bit to see why those might even be necessary.  It would appear the @override will inherit regardless so seems like those are a waste.  Unfortunately that plugin doesn't seem to clean itself up if settings are changed so that becomes a manual effort.  Being that I added this, if I learn those are unnecessary and since they don't help checkstyle issues, then I'll take the time to get them pruned.  I'll try to get back to this soon.